sequencer/api/reward.toml

```diff
@@ -8,3 +8,9 @@ PATH = ["reward-balance/:height/:address"]
 ":address" = "Literal"
 ":height" = "Integer"
 DOC = "Get balance in reward state at a specific height. Expected parameters are height which is an integer and an Ethereum address in hex format"
+
+[route.get_reward_account_proof]
```

Comment:
IIUC currently the client will get a proof for the V1 or the V2 rewards merkle tree but doesn't have an easy way to check what it is. Since we don't need this to work for the V1 tree maybe it would be simpler to only enable the API for V2? 

Wdyt?

Reply:
if we only need v2 proof then we can call v2 endpoint i.e `reward-state-v2/`?

Reply:
@imabdulbasit reminded me that these two versions are served on different API versions so my comment is moot.